custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -84,6 +85,49 @@ func (t *Translator) GetRespForPod(response *stackdriver.ListTimeSeriesResponse,
 	return nil, fmt.Errorf("Illegal state")
 }
 
+// GetSDReqForPods returns Stackdriver request for query for multiple pods.
+func (t *Translator) GetSDReqForPods(podList runtime.Object, metricName string) (*stackdriver.ProjectsTimeSeriesListCall, error) {
```

Comment:
What do you mean? Do you want me to pass here the list of pod IDs?

The reasoning I had in mind was to put as much logic here as opposed to provider.go, because only the translator is unit tested. When you look at function: GetRespForPods, there is another reason for passing runtime.Object - I use there both pod names and IDs, so I would have to create some other structure to pass this as an argument.

What I don't like here is that pods/podLists are not typed. I agree it makes more sense to convert podList and pod to typed objects as soon as possible, i.e. in provider.go. I decided that here it's not worth it. WDYT?